lib/api/apiUtils/object/objectLockHelpers.js

```diff
@@ -0,0 +1,64 @@
+const { errors } = require('arsenal');
+/**
+ * Calculates retain until date for the locked object version
+ * @param {object} retention - includes days or years retention period
+ * @return {object} the date until the object version remains locked
+ */
+function calculateRetainUntilDate(retention) {
+    const { days, years } = retention;
+    const date = new Date();
+    // Calculate the number of days to retain the lock on the object
+    const retainUntilDays = days || years * 365;
+    const retainUntilDate
+        = date.setDate(date.getDate() + parseInt(retainUntilDays, 10));
```

Comment:
do you need the parseInt here?

Reply:
I don't really - it can never be float so parseInt is kinda excessive; gonna remove it.